src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -39,7 +39,7 @@ export const CheckImg = styled.img`
   }
 
   @media (min-width: 961px) {
-    width: 20vw;
+    width: 100px;
     height: auto;
     bottom: 0;
     right: 50vw;
```

Comment:
this `bottom` and `right: 50vw` is not doing anything since the element position is "static". Firefox says to me that it's not being applied to the element. Then I think can be removed

![imagen](https://user-images.githubusercontent.com/66505715/158848006-63b3e766-98b6-4f71-aa9e-3fc108f0e196.png)